### DISCUSSION TITLE
Need Copilot extension on Open VSX Registry

### DISCUSSION THREAD

**Discussion by Ashesh** [community/community#58566] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

A large bunch of our developers use Texas Instrument's Code Composite Studio (https://www.ti.com/tool/CCSTUDIO) which only serves VS Code extensions from Eclipse's Open VSX Registry - https://open-vsx.org/user-settings/extensions 

Could you upload your VS Code Copilot Extension to this Open-source registry.

**Comment by Ashesh** [comment]
Please go through this to understand how Copilot plugin for VS Code can be published for use from Open VSX Registry https://github.com/eclipse/openvsx/wiki/Publishing-Extensions

**Comment by ananta-dev** [comment]
I work in the Data Science team of a large insurance company. We use VS Code on Posit Workbench, which only allows installing extensions from Open VSX marketplace. We have Github Copilot licenses but we cannot them it because there is no Github Copilot extension on Open VSX.

I am sure Github would act on this if they realised there are paying customers that cannot use the service due to the lack of the required extension in some extension repositories. 

Is this the best place for this request?

**Comment by davidrsch** [comment]
This should be done soon, or so I hope. Since copilot has been announced to be open source in a near future.
